### ISSUE TITLE
AfterCast not working on Classic Era

### ISSUE THREAD

**Issue by svenbledt** [Rottenbeer/ItemRack#295]
First of all, arg2 is spell GUID wich is actualy not a option i think that ppl enter the spellguid,

arg3 wich should then be spellid is fine for me but the event does just nothing and i think it doesnt work at all on Classic Era.

Pls fix this its needed for warriors and would be highly appreciated.

**Comment by DedodedoMinion** [comment]
Yeah I had this issue as well, you can do a workaround for it by defining the args at the top as they aren't passed in the loadstring
```lua
local arg1,arg2,arg3 = ...
```
this will make arg3 the spellID

here's an example that I use to equip my stealth detction set when casting track hidden

```lua
local arg1,arg2,arg3 = ...
local spell = 19885
local set = "Stealth Detect"
if arg1=="player" and arg3==spell then
  EquipSet(set)
end
```

**Comment by Alessandro-Barbieri** [comment]
For diamond flask you can use Buff Gained instead
